firmware/ventilator-controller-stm32/Core/Src/Pufferfish/Driver/Serial/Nonin/Sensor.cpp

```diff
@@ -30,16 +30,16 @@ InitializableState Sensor::output(float &spo2, float &hr) {
     return InitializableState::failed;
   }
 
-  if (device_.output(measurements_) == Device::PacketStatus::available) {
+  if (device_.output(measurements_) == PacketStatus::ok) {
```

Comment:
Do we need to add a TODO comment about handling the case where the `Device::output` method doesn't return `ok`? Right now it seems that `Sensor::output` returns `ok` regardless of what happens. Maybe just what we need to change in order to add POST checks for the Nonin sensor? (there's probably other things we'll need to change, too).

This is probably code I wrote in a hurry. If I had used a return-early structure (i.e. `status = device_.output(...); if (status !== ok) {return an error code or do nothing but have a TODO comment for dealing with this} `), this would have been easier to catch and understand.